contribs/emissions/src/main/java/org/matsim/contrib/emissions/WarmEmissionAnalysisModule.java

```diff
@@ -307,22 +304,19 @@ void throwWarmEmissionEvent(double leaveTime, Id<Link> linkId, Id<Vehicle> vehic
 
 				// compute faction.  This cannot be done earlier since efkey.component is needed.
 				fractionStopGo = getFractionStopAndGo(freeVelocity_ms * 3.6, averageSpeed_kmh, vehicleInformationTuple, efkey);
-				logger.info("fractionStopGo is: " + fractionStopGo);
 
 				double efStopGo_gpkm = 0.;
 				if (fractionStopGo > 0) {
 					// compute emissions from stop-go fraction:
 					efkey.setTrafficSituation(STOPANDGO);
 					efStopGo_gpkm = getEf(vehicleInformationTuple, efkey).getFactor();
-					logger.warn("pollutant=" + warmPollutant + "; efStopGo=" + efStopGo_gpkm);
 				}
 
 				double efFreeFlow_gpkm = 0. ;
 				if ( fractionStopGo<1.) {
 					// compute emissions for free-flow fraction:
 					efkey.setTrafficSituation(FREEFLOW);
 					efFreeFlow_gpkm = getEf(vehicleInformationTuple, efkey).getFactor();
-					logger.warn("pollutant=" + warmPollutant + "; efFreeFlow=" + efFreeFlow_gpkm);
```

Comment:
Same here: I would prefer setting it to logger.debug(...) instead of removing it.

Reply:
Will change it.